### ISSUE TITLE
converter for MetaMorpheus output to MSstatsPTM 

### ISSUE THREAD

**Issue by ofleitas** [Vitek-Lab/MSstatsPTM#48]
Hello 

Could be possible to add a dedicated converter for MetaMorpheus output in MSstatsPTM?

Best regards,

**Comment by tonywu1999** [comment]
@ofleitas 

Hello,

I'll be working on adding a converter for Metamorpheus output into MSstatsPTM.  To assist us with implementation, could you provide sample datasets, i.e. one Metamorpheus dataset from a regular label free experiment and one Metamorpheus dataset from a PTM experiment?

If you have privacy restrictions for your dataset, you could anonymize the proteins, the conditions, etc.  We'd also be happy to work with a subset of your dataset.  You can find an example at [SampleMetamorpheus.zip](https://github.com/Vitek-Lab/MSstatsPTM/files/13651894/SampleMetamorpheus.zip)

Thanks,
Tony

**Comment by ofleitas** [comment]
Hello

Great that you going to implement a converter for metamorpheus output into
MSstatsPTM. The Metamorpheus output data that I have is from the global PTM
discovery (G-PTM-D) strategy. Does this data work for you?

**Comment by tonywu1999** [comment]
Yes, from my understanding, output data from the global PTM discovery (G-PTM-D) strategy should work.  

Thanks.

>Hello
>
> Great that you going to implement a converter for metamorpheus output into
MSstatsPTM. The Metamorpheus output data that I have is from the global PTM
discovery (G-PTM-D) strategy. Does this data work for you?

**Comment by ofleitas** [comment]
Can I share it with you by drive?  The data is too big to share here.

**Comment by tonywu1999** [comment]
Yes, you can share by drive.

**Comment by tonywu1999** [comment]
@ofleitas 

Hi, I published code to github for the Metamorpheus converter (function name is `MetamorpheusToMSstatsPTMFormat`).  Could you try it out and provide any feedback / difficulties you encounter?  You can install the code using the command `devtools::install_github("Vitek-Lab/MSstatsPTM", build_vignettes = TRUE)` on RStudio.

In the meantime, our team will be performing beta testing on the converter before pushing to Bioconductor.

**Comment by ofleitas** [comment]
Great news!!!!!
 Sorry for the late reply. At the moment it will be almost impossible for
me to test but as soon as I have some free time I will do it.
Best regards,
Osmel
